src/js/components/help/helpContent.jsx

```diff
@@ -105,14 +105,6 @@ export default class HelpContent extends React.Component {
                     with Treasury staff, please email
                     <a href="mailto:DATAPMO@fiscal.treasury.gov">DATAPMO@fiscal.treasury.gov</a>.
```

Comment:
There is a space missing before this link. Unrelated to your changes, but an easy fix.